### DISCUSSION TITLE
How to load Sandpack locally only?

### DISCUSSION THREAD

**Discussion by tomasts248** [codesandbox/sandpack#1013] [Q&A]
I noticed using the react version that an iframe is created that it calls to Codingsandbox `https://2-7-1-sandpack.codesandbox.io`/, therefore is there a way to completely host Sandpack locally? Perhaps I'm just getting it wrong..

**Comment by reidbarber** [comment]
https://sandpack.codesandbox.io/docs/guides/hosting-the-bundler
